docs/data-sources/aviatrix_vpc.md

```diff
@@ -57,3 +57,5 @@ In addition to all arguments above, the following attributes are exported:
   * `cidr` - Private subnet CIDR.
   * `name` - Private subnet name.
   * `subnet_id` - Private subnet ID.
+* `availability_domains` - List of OCI availability domains.
```

Comment:
Should we update the OCI examples in the gateway documentation to show the use of these values?